app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomActivity.kt

```diff
@@ -86,9 +86,11 @@ class ChatRoomActivity : AppCompatActivity(), HasSupportFragmentInjector {
 
         isChatRoomSubscribed = intent.getBooleanExtra(INTENT_CHAT_IS_SUBSCRIBED, true)
 
-        addFragment("ChatRoomFragment", R.id.fragment_container) {
-            newInstance(chatRoomId, chatRoomName, chatRoomType, isChatRoomReadOnly, chatRoomLastSeen,
-                    isChatRoomSubscribed)
+        if (supportFragmentManager.findFragmentByTag("ChatRoomFragment") == null) {
```

Comment:
Next time we'll need to add a `const` with the tag for our fragments.

Reply:
Thanks for letting me know, I will remember for next time.